sass/_components.scss

```diff
@@ -2151,6 +2151,12 @@
         height: 100%;
     }
 
+    .button {
+        -webkit-appearance: none;
+        -moz-appearance: none;
+        appearance: none;
```

Comment:
You should just be able to use appearance here - the autoprefixer module will add the -webkit- and -moz- variants

Reply:
Ahh yeah I forgot we had the autoprefixer. Cheers!